src/app/api/me/route.js

```diff
@@ -10,14 +10,14 @@ import cleanerData from '~/app/api/libs/cleanerData'
 
 export const GET = async request => {
   try {
-    const userId = authenticateToken(request)
+    const { userId } = authenticateToken(request)
     const payload = await queryDB({
       entity: 'user',
       queryType: 'findUnique',
       filter: { id: Number(userId) },
       includes: ['permissions']
     })
-    const hasPermission = getPermissionsByEntity({ permissions: payload.permissions, entity: Me, action: 'findUnique' })
+    const hasPermission = getPermissionsByEntity({ permissions: payload.permissions, entity: Me, action: 'FINDUNIQUE' })
```

Comment:
We should use SCREAMING_SNAKE_CASE
```suggestion
    const hasPermission = getPermissionsByEntity({ permissions: payload.permissions, entity: Me, action: 'FIND_UNIQUE' })
```